.flowconfig

```diff
@@ -2,21 +2,11 @@
 0.61.0
 
 [ignore]
-.*test
-./build
+.*/node_modules/jsxhint/.*
 
 [include]
-./lib
-index.js
 
 [libs]
 interfaces/
 
 [options]
-suppress_comment=\\(.\\|\n\\)*\\s*\\$FlowFixMe.*
```

Comment:
I like keeping these around in case we need to use them later on; Flow is behaving well now but won't always

Reply:
FlowFixMe is works out of the box. Another errors I always try to solve.